types/artifacts.d.ts

```diff
@@ -431,17 +431,10 @@ declare global {
         isPicture: boolean;
         /** Flags whether this element was contained within a ShadowRoot */
         isInShadowDOM: boolean;
-        /** Flags whether this element was sized using a non-default `object-fit` CSS property. */
-        usesObjectFit: boolean;
-        /** Flags whether this element was rendered using a pixel art scaling method.
-         *  See https://developer.mozilla.org/en-US/docs/Games/Techniques/Crisp_pixel_art_look for
-         *  details.
-         */
-        usesPixelArtScaling: boolean;
-        /** Flags whether the image has a srcset with density descriptors.
-         *  See https://html.spec.whatwg.org/multipage/images.html#pixel-density-descriptor
-         */
-        usesSrcSetDensityDescriptor: boolean;
+        /** `object-fit` CSS property. */
+        ObjectFit: string;
```

Comment:
```suggestion
        cssComputedObjectFit: string;
```

WDYT to match the other `cssComputed*` property convention?

Reply:
I like this! Makes a lot more sense and matches other properties 👍